### PR TITLE
chore(grid): grid refactor

### PR DESCRIPTION
**Pull request by sharkparty**
**Before submitting a pull request,** please make sure the following is done:

I have done **all** of the following:

- [x] Added a top level class to all my components `'.anchor-[COMPONENT NAME]'`.
- [x] Used [conventional commits](https://www.conventionalcommits.org) for all work.
- [x] Tested my solution on Mobile & Tablet.
- [x] Wrote [unit tests](https://jestjs.io/docs/en/getting-started) for states and all behavior (`npm test`) and passed coverage thresholds.
- [x] Updated snapshots for all permutations (`npm test -- -u`).
- [x] Accounted for hover, focus, blur, visited, & error states because they are not edge cases.
- [x] Created TODOs for known edge cases.
- [x] Documented all of my changes (inline & doc site).
- [x] Made sure that all accessibility errors are resolved.
- [x] Added [stories](https://storybook.js.org/docs/basics/introduction/) with knobs for all possible configurations.
- [x] De-linted and ran [prettier](https://github.com/prettier/prettier) (`npm run pretty`) on my code.
- [x] Added name to OWNERS file for all new components
- [x] If adding a new component, add its export to the rollup config

---------
**Outline your feature or bug-fix below**

My code adds the new grid to our library.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@retailmenot/anchor",
-  "version": "0.0.1-beta.12",
+  "version": "0.0.1-beta.13",
```
Comment (sharkparty) [inline package.json]: Comment testing GH integration.

**Comment by zero0Halo** [comment]
This effectively disables `Layout`, is it being deprecated or are you setting it up for a refactor?

File: src/Grid/Grid/Grid.component.tsx [hunk 2]
```diff
@@ -1,58 +1,10 @@
...
-);
+export {
+    Col,
+    Visible,
+    Row,
+    Container,
+    ScreenClass,
+    ScreenBadge,
+    Hidden,
+} from 'react-awesome-styled-grid';
```
Comment (zero0Halo) [inline src/Grid/Grid/Grid.component.tsx:13]: We should give these Anchor classname's if nothing else. Pass props, etc?

Comment (sharkparty) [reply src/Grid/Grid/Grid.component.tsx:13]: Solid point. I'll also be refactoring the docs site in this PR, so I'll add that to my list. I wanted to get this up early and get everyone in the loop to understand the API, the lib we're padding through, and make any concerns known as this evolves.

File: src/Layout/Page/Page.spec.tsx [hunk 3]
```diff
@@ -2,17 +2,15 @@
...
 import {
     Page,
-    DEFAULT_HEADER_HEIGHT,
-    DEFAULT_FOOTER_HEIGHT,
 } from './Page.component';
 
 describe('Component: Page', () => {
     const subject = <Page />;
-    const wrapper = mount(subject);
+    // const wrapper = mount(subject);
```
Comment (brandonlilly) [inline src/Layout/Page/Page.spec.tsx:11]: hmm

File: src/Layout/SidebarLayout/SidebarLayout.component.tsx [hunk 4]
```diff
@@ -93,29 +92,27 @@ export const SidebarLayout = ({
...
-                        <Cell width={2}>{children}</Cell>
-                    </CustomAdaptor>
-
-                    <CustomAdaptor minWidth={StandardBreakpoints.md.min}>
-                        <Cell>{sidebar}</Cell>
-                        <Cell>{children}</Cell>
-                    </CustomAdaptor>
-                </Grid>
-            )}
+            {/*{sidebarAlign === RIGHT ? (*/}
```
Comment (brandonlilly) [inline src/Layout/SidebarLayout/SidebarLayout.component.tsx:95]: Preserving for when it's reimplemented?

**Review by brandonlilly — APPROVED** [review]
Assuming we're going to fix up some of these things things in a subsequent PR

File: src/Layout/SidebarLayout/SidebarLayout.component.tsx (hunk 4 above)
Comment (sharkparty) [reply src/Layout/SidebarLayout/SidebarLayout.component.tsx:95]: Yeah, I was going to just delete the entire layout directory until we decide what a non RMN layout implementation looks like, but I didn't want to cull this repo that hard when likely we'll want to bring it back soon.

File: src/Layout/Page/Page.spec.tsx (hunk 3 above)
Comment (sharkparty) [reply src/Layout/Page/Page.spec.tsx:11]: 👎

Comment (sharkparty) [reply src/Layout/Page/Page.spec.tsx:11]: Buttt, the tests that were here only tested the presence of the grid, which snapshots do so they weren't tests we needed anyway.

**Pull request merged**